apps/evm/src/app/[lang]/(bridge)/stake/components/EvmStrategy/EvmStrategyForm.tsx

```
@@ -0,0 +1,269 @@
+import { BridgeTransaction } from '@/types';
+import { calculateAmountUSD } from '@/utils';
+
+const symbols = ['WBTC', 'SolvBTC', 'SolvBTC.BBN', 'LBTC'];
+const initialSymbol = 'WBTC';
+
+// eslint-disable-next-line @typescript-eslint/ban-types
+type EvmStrategyFormProps = {
+  strategy?: StrategyData;
+};
+
+const EvmStrategyForm = ({}: EvmStrategyFormProps): JSX.Element => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)